docker-compose.yml

```diff
@@ -1,31 +1,17 @@
 version: '3.4'
```

Comment:
The lock file is not up to date with the latest changes in composer.json

Reply:
phpspec/prophecy 1.12.2 requires php ^7.2 || ~8.0, <8.1 -> your php version (8.1.10) does not satisfy that requirement.

Reply:
I have removed composer.lock from PR and put it in gitignore/ file. Do you think i should revert and add it in PR ?